docs/dev/relay_bring_your_own_codegen.rst

```diff
@@ -18,7 +18,7 @@
 =============================
 Bring Your Own Codegen To TVM
 =============================
-**Author**: `Zhi Chen <https://github.com/zhiics>`_, `Cody Hao Yu <https:://github.com/comaniac>`_
+**Author**: `Zhi Chen <https://github.com/zhiics>`_, `Cody Yu <https://github.com/comaniac>`_
```

Comment:
Would be great if we can remove the authors, we tried to keep the names in tutorials, but avoid doing so in the docs, to encourage more people to collaborate on these documents. The github commit history still shows the contribution